app/models/user.rb

```diff
@@ -6,7 +6,60 @@ class User < ApplicationRecord
 
   validates :name, presence: true, length: { maximum: 20 }
 
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i.freeze
```

Comment:
- [ ] Devise already handles validating emails. Check `config/initializers/devise.rb` file, there is `config.email_regexp = /\A[^@\s]+@[^@\s]+\z/` option.

Reply:
Hey @mosaaleb, thanks for the review and the great feedback. We have implemented the change you requested.